internal/extsvc/azuredevops/client.go

```diff
@@ -78,16 +82,46 @@ func (c *Client) ListRepositoriesByProjectOrOrg(ctx context.Context, opts ListRe
 	}
 
 	var repos ListRepositoriesResponse
-	if _, err = c.do(ctx, req, &repos); err != nil {
+	if _, err = c.do(ctx, req, "", &repos); err != nil {
 		return nil, err
 	}
 
 	return repos.Value, nil
 }
 
+// AzureServicesProfile is used to return information about the authorized user, should only be used for Azure Services (https://dev.azure.com)
+func (c *Client) AzureServicesProfile(ctx context.Context) (Profile, error) {
+	qs := make(url.Values)
+
+	qs.Set("api-version", apiVersion)
+
+	urlProfile := url.URL{Path: "/_apis/profile/profiles/me", RawQuery: qs.Encode()}
+
+	req, err := http.NewRequest("GET", urlProfile.String(), nil)
+	if err != nil {
+		return Profile{}, err
+	}
+
+	var p Profile
+	//The endpoint is https://vssps.dev.azure.com
```

Comment:
```suggestion
	//The endpoint is https://app.vssps.visualstudio.com
```
or maybe this comment can be deleted? it is pretty clear from the line below